<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>밴드 채팅 보내기 매크로</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="keyword_label">
          <property name="text">
           <string>키워드</string>
          </property>
          <property name="buddy">
           <cstring>keyword_edit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="keyword_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="find_btn">
        <property name="text">
         <string>찾기</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="content_group">
      <property name="title">
       <string>내용</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="content_edit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="run_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>실행</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>중지</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>find_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>find()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>run_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stop_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>content_edit</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_text_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>664</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>find()</slot>
  <slot>run()</slot>
  <slot>stop()</slot>
  <slot>on_text_changed()</slot>
 </slots>
</ui>
